src/app/reviews/reviews.component.scss

```diff
@@ -0,0 +1,25 @@
+
+.empty-status {
+  background: var(--ion-color-primary-contrast);
```

Comment:
there's a class in `style.scss` called `background-white`. Remove the CSS here and use this class instead